### DISCUSSION TITLE
Team B - Course 1

### DISCUSSION THREAD

**Discussion by mindgitrwx** [lablup/contribution-academy-2021#42] [Show and tell]
Backend.AI 사용해보기

**Comment by AlchemistX** [comment]
## Backend.AI 사용해보기
### Executed Time (for traning only)
![ExecutedTime](https://user-images.githubusercontent.com/1624251/129021800-bd694243-daa1-4e29-85f5-ed828c10a5f3.png)

### Accuracy
![Accuracy](https://user-images.githubusercontent.com/1624251/129021815-4be70bc7-6d79-46ef-bfb4-9ddafc908fca.png)

### Total allocated resources: 2 CPU Core/6 GB RAM/0 FGPU/1 Session
![Resource](https://user-images.githubusercontent.com/1624251/129021826-2cad4d9b-734e-4263-9b59-4452834db9e7.png)

**Comment by TeusEE** [comment]
Course 1 Backend.ai 사용해보기

-. executed time (for training only)
![image](https://user-images.githubusercontent.com/79349894/129024133-d94e8b23-ae73-41e4-b82a-97ea247f0605.png)
train 총 소요시간은 71초5ms 소요되었습니다.
(Nvidia GPU Cloud문제인지는 모르겠지만, 소요시간이 엄청 오래걸렸습니다)
--> 일반 환경에서는 정상적으로 1epoch당 3초 정도가 소요되었습니다
![image](https://user-images.githubusercontent.com/79349894/129029845-ee5c00a6-b476-42ed-ac13-50d22884fa60.png)


-. accuracy
![image](https://user-images.githubusercontent.com/79349894/129024336-780ecd6b-a640-4d87-9fdd-f106d117d015.png)
test data-set 기준으로 0.8746의 분류정확도를 보여줍니다.

total allocated resources:
![image](https://user-images.githubusercontent.com/79349894/129024463-62a32c91-be6e-4588-8127-9e9f2f66b9f0.png)
테스트에 사용된 리소스입니다.

==============================================================================
#이후 Nvidia GPU 클라우드 말고 일반 환경에서 실행한 추가 테스트입니다.--
추가적으로 간단하게 합성곱 신경망을 써서 아래와 같은 딥러닝 모델을 생성했으며
![image](https://user-images.githubusercontent.com/79349894/129027810-fab3f14c-50fd-4d7c-8f2c-708cd7f37caa.png)
![image](https://user-images.githubusercontent.com/79349894/129028045-563646a7-92f0-47cd-9992-e4cb50d828df.png)

모델 학습 소요시간과, 분류 정확도는 아래와 같습니다(DNN보다... 당연하지만 좋습니다)
![image](https://user-images.githubusercontent.com/79349894/129028155-5f239a12-4c4d-47b0-94fa-2a24c2c93fdd.png)
![image](https://user-images.githubusercontent.com/79349894/129028188-95163d43-52e3-4ab3-bd65-234abf4e7f43.png)

**Comment by min1826** [comment]
* **executed time**
<img width="700" alt="1" src="https://user-images.githubusercontent.com/71456233/129028663-0c7152ce-60f6-4d5f-85eb-66dba8adf4cb.png">

* **accuracy**
<img width="350" alt="2" src="https://user-images.githubusercontent.com/71456233/129028723-17c0e32c-9b26-46c4-bcf8-0040b2ee68d4.png">

* **total allocated resources**
<img width="700" alt="3" src="https://user-images.githubusercontent.com/71456233/129028897-a25a89c0-2233-4c15-96ce-b0007714035b.png">

**Comment by mindgitrwx** [comment]
![image](https://user-images.githubusercontent.com/37054916/129041122-3b2567f1-925c-45ae-8bba-9b7a6dc66e8d.png)

![image](https://user-images.githubusercontent.com/37054916/129041206-d47e7bbb-5767-47d2-9da2-080905a90b2d.png)

많은 시도를 해 보았으나, 군대의 사이버지식정보방의 환경에서는 포트를 차단하는 것으로 보입니다. (ERR_CONNECTION_TIMED_OUT)
![image](https://user-images.githubusercontent.com/37054916/129040620-82a9f3e8-8e02-4e3d-8b83-2ce284cc54b3.png)

**Comment by mindgitrwx** [comment]
@lizable 

B팀은 저 빼고 모두 완료되었습니다.

(확정적으로 8월 16일부터 여기를 벗어나 원래 제 환경에서 개발이 가능합니다)

--------------------------------------
I executed it on my own desktop. 

8.17
![course1_Feed_the_model](https://user-images.githubusercontent.com/37054916/129737752-fab4b666-8306-4e60-b147-dc1117478bc4.png)
![course1_evaluate_accuracy](https://user-images.githubusercontent.com/37054916/129737748-d69e20a0-3786-4c6e-a3c6-489783581fea.png)
